src/screens/AirdropDashboard/AirdropDashboardScreen.tsx

```diff
@@ -0,0 +1,104 @@
+import { CompositeNavigationProp } from '@react-navigation/native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { FC } from 'react';
+import { View, StyleSheet, Text, TouchableOpacity } from 'react-native';
+
+import { images } from 'app/assets';
+import { Header, ScreenTemplate, Button, Image, Countdown } from 'app/components';
+import { CONST, MainCardStackNavigatorParams, Route, RootStackParams } from 'app/consts';
+import { getFormattedAirdropDate } from 'app/helpers/airdrop';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../loc');
+
+interface Props {
+  navigation: CompositeNavigationProp<
+    StackNavigationProp<RootStackParams, Route.MainCardStackNavigator>,
+    StackNavigationProp<MainCardStackNavigatorParams, Route.AirdropDashboard>
+  >;
+}
+
+export const AirdropDashboardScreen: FC<Props> = ({ navigation }) => (
+  <ScreenTemplate
+    footer={
```

Comment:
Since it's a quite big JSX block, maybe consider extracting it to `renderFooter()` method to increase the overall readability?

Reply:
it's not a good idea because you will create renderFooter function on every rerender, a better solution is to move it to a separate component